definitions/checks/disk/performance.rb

```diff
@@ -38,11 +42,19 @@ def check_only_single_device?
 
       def dirs_to_check
         return DEFAULT_DIRS.first(1) if check_only_single_device?
+
         DEFAULT_DIRS
       end
 
       private
 
+      def warning_message
+        "\nWARNING: Before taking any updates make sure"\
```

Comment:
Alternative wording:

```
WARNING: Low disk speed might have negative impact on the system.
Please consult https://access.redhat.com/solutions/3397771 before proceeding
with further actions.
``

Or even better, how about @swadeley having a look and giving us a native speaker suggestion on how to best formulate the message :)

Reply:
WARNING: Low disk speed might have a negative impact on the system. See https://access.redhat.com/solutions/3397771 before proceeding.

Reply:
Short and clean. Thanks @swadeley